applications/areg-portal/frontend/src/components/icons.tsx

```diff
@@ -264,3 +264,53 @@ export const DescSortedIcon = (props: SvgIconProps) => (
     />
   </SvgIcon>
 );
+
+export const FaqIcon = (props: SvgIconProps) => (
+  <SvgIcon
+    sx={{ height: "0.75rem", width: "0.875rem " }}
```

Comment:
A better way to set the default dimension is to change the viewbox. Specifying size here is not ideal